settings/global.mdx

```diff
@@ -607,10 +607,15 @@ Example:
     <ResponseField name="indexHiddenPages" type="boolean" default="false">
     Enables indexing pages not included in `navigation`.
     </ResponseField>
+    <ResponseField name="noindex" type="boolean" default="false">
+    Disables search engine indexing in your given MDX file.
+    </ResponseField>
```

Comment:
i don't believe this exists in the mintconfig today, iirc you have to specify `noindex` per-page in the frontmatter?

Reply:
yep mayank is correct here